scoreboard/attachments/__init__.py

```diff
@@ -40,11 +41,23 @@ def get_backend_type():
     backend = get_backend_path()
     return urlparse.urlparse(backend).scheme
 
+def patch(_backend_type):
```

Comment:
This feels very repetitive.  We could have a method (such as "get_backend"), that does the comparison/import, then returns `backend`?  This way we don't have to repeat the if statements/imports